### ISSUE TITLE
[1.4.2] Installation issues to fix

### ISSUE THREAD

**Issue by craigh** [zikula/core#2840]
from http://zikula.org/forums/topic/59974

| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.2
| PHP Version       | 5.x unknown

### Expected behavior

full install with no errors

### Actual behavior
 - [x] Needed to install Apache module access_compat to process order commands in htaccess files. Without this no javascript or styles loaded into install page.
 - [x] Link on installation to documentation on install page didn't work. Page looked for INSTALL-1.4.0.MD. Changed the name of the file in docs/en to match and then link worked.
 - [x] Installation progressed quickly through all steps except the last (Install assets) which reported that it failed. Then could not do login. Error was "Data too long for column 'sessid' at row 1" in table session_info. I changed the length of this field from 40 to 60. I think the session id was 52 characters long. Login then succeeded and installation appeared complete.

**Comment by craigh** [comment]
PHP Version 5.5.14
Apache 2.4.16
OpenSuse distribution Leap 42.1 in Virtual Box

**Comment by craigh** [comment]
refs #2938

**Comment by cmfcmf** [comment]
> Needed to install Apache module access_compat to process order commands in htaccess files. Without this no javascript or styles loaded into install page.

Here is an example of a Symfony `.htaccess` file: https://github.com/symfony/symfony-standard/blob/master/src/.htaccess
```htaccess
<IfModule mod_authz_core.c>
    Require all denied
</IfModule>
<IfModule !mod_authz_core.c>
    Order deny,allow
    Deny from all
</IfModule>
```

**Comment by craigh** [comment]
therefore my comment above:
>refs #2938

you are saying that changing the .htaccess file like that would also solve the `access_compat` issue?

**Comment by cmfcmf** [comment]
> you are saying that changing the .htaccess file like that would also solve the access_compat issue?

No, just wanted to add a reference to Symfony. I haven't actually looked in depth and I don't know whether this fixes the actual problem.

**Comment by Guite** [comment]
First point (#2938) fixed in #2982

**Comment by Guite** [comment]
Second point fixed in #2983

**Comment by craigh** [comment]
thanks @Guite nice work.
